src/monitor.rs

```diff
@@ -113,6 +113,14 @@ pub trait MonitorHandleProvider: AsAny + fmt::Debug + Send + Sync {
     #[cfg_attr(not(web_platform), doc = "detailed monitor permissions.")]
     fn position(&self) -> Option<PhysicalPosition<i32>>;
 
+    /// Returns physical size of monitor in millimeters.
```

Comment:
Could I get you to provide an example of where you might want to use this vs. `scale_factor`?

Is it to e.g. show an accurate ruler?

Reply:
I think it could be used for VR, CAD, eye tracking, some medical applications, flight simulators, and so on. Basically anything that requires some kind of calibration with real world. My use case is that I've been developing a hobby web browser and want CSS mm to be equivalent to real mm 😁.

Reply:
Maybe add these examples to the docs?

Reply:
> My use case is that I've been developing a hobby web browser and want CSS mm to be equivalent to real mm 😁.

But that'd be out of spec, right?

Reply:
Yeah, it won't be compatible, at least not completely.

Reply:
> Maybe add these examples to the docs?

I don't think that this is relevant to add. In my opinion API documentation should only contain a description of how this function works. Such examples in description of small utility function would look out of place.

Reply:
Fair enough.